### PR TITLE
Choose exact transition match if available

### PR DESCRIPTION
**Pull request by pcockwell**
Addresses issue #281 by attempting to choose the exact match if possible.

I am reasonably new to `go`, but believe I have implemented this correctly. That being said, I was unable to determine where the tests for this are written, as well as how to execute those tests. Please feel free to request or make changes as needed, though if you want me to do them you'll have to provide some detail in your instructions so I can understand what you need.

### CODE REVIEW

File: jiradata/TransitionsFuncs.go [hunk 1]
```diff
@@ -9,10 +9,17 @@ import (
 // or nil
 func (t Transitions) Find(name string) *Transition {
 	name = strings.ToLower(name)
+	matches := []Transitions{}
 	for _, trans := range t {
-		if strings.Contains(strings.ToLower(trans.Name), name) {
+		if strings.Compare(strings.ToLower(trans.Name), name) == 0 {
```
Comment (coryb) [inline jiradata/TransitionsFuncs.go]: this is more commonly written as `if strings.ToLower(trans.Name) == name {`

Comment (pcockwell) [reply jiradata/TransitionsFuncs.go]: Done!

**Comment by coryb** [comment]
not sure what is up with the tests, looks unrelated to your changes, I will look into that when I get a chance.

**Comment by pcockwell** [comment]
@coryb Any idea how long this will be before it makes it into a new release? Just curious so I can check back when ready

**Pull request merged**